corehq/apps/cloudcare/views.py

```diff
@@ -187,11 +194,21 @@ def get(self, request, domain):
     def get_option_apps(self, request, domain):
         restore_as, set_cookie = self.get_restore_as_user(request, domain)
         apps = self.get_web_apps_available_to_user(domain, restore_as)
+        accessible_apps = []
```

Comment:
It looks like the two places you updated are the only usages of `get_web_apps_available_to_user`, so why not move this check into that function?